### PR TITLE
Removing Unneeded Dependencies

### PR DESCRIPTION
**Pull request by phillipskevin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,6 @@
     "concat-stream": "^1.5.1",
     "coveralls-send": "0.0.2",
     "cross-spawn-async": "^2.1.9",
-    "documentjs": "^0.4.1",
```
Comment (matthewp) [inline package.json]: i'm assuming this is going to be done with a plugin, is there an existing issue for that?

File: package.json [hunk 2]
```diff
@@ -55,9 +54,7 @@
     "debug": "^2.2.0",
     "mkdirp": "^0.5.1",
     "q": "^1.4.1",
-    "steal-platform": "0.0.4",
-    "yeoman-environment": "^1.2.7",
-    "yeoman-generator": "^0.22.3"
```
Comment (matthewp) [inline package.json]: the generator isn't needed?

File: package.json (hunk 1 above)
Comment (phillipskevin) [inline package.json]: Unless there's something I'm missing, this didn't do anything. `documentjs` is still added as a devDependency of projects created with `donejs add app` by this line: https://github.com/donejs/cli/blob/major/package.json#L82.

File: package.json (hunk 2 above)
Comment (phillipskevin) [inline package.json]: No, I don't think so. It's not required to use `yeoman-environment` and we're not using `yeoman-generator` anywhere.

File: package.json (hunk 1 above)
Comment (matthewp) [inline package.json]: right 👍

**Pull request merged**